### PR TITLE
Updating translations

### PR DESCRIPTION
**Pull request by Baklap4**
### What does this PR do?

* Makes the template pot file consistent; eg: Capitals, sentences in menus ending with dots. Upper and lowercase words etc.
* Provide translations and aim for 100% coverage:

- [x] 100% English - Code generated
- [x] 100% Dutch (@Baklap4)
- [x] 100% French (@OznOg )
- [x] 100% Spanish (@Lord-Kamina )
- [x] 100% German (@twollgam ?)
- [ ] 100% Slovak (@jose1711 ? )
- [ ] 100% Hungarian (@skyli91 ?)
- [ ] 100% Japanese (@nieknooijens ?)
- [ ] 100% Chinese
- [ ] 100% Portuguese
- [ ] 100% Finnish
- [ ] 100% Danish
- [ ] 100% Swedish
- [ ] 100% Asturian
- [ ] 100% Persian
- [ ] 100% Italian
- [ ] 100% Polish

If you can't commit to this branch please make a git patch and i'll make sure it gets in here 👍

Progress on the languages can be found on the wiki: https://github.com/performous/performous/wiki/Translation#overview i'll make sure to update it after every commit to this branch.

### Closes Issue(s)

None

### Motivation

Inconsistency.

### CODE REVIEW

File: data/config/schema.xml [hunk 1]
```diff
@@ -142,7 +142,7 @@ to save the current settings to XML.
 	</entry>
 	<entry name="game/drum_promode" type="bool" value="true">
 		<short>Drums use Pro mode</short>
-		<long>Drums use Pro mode (toms are not cymbals)</long>
+		<long>Drums use Pro mode (toms are not cymbals).</long>
```
Comment (Lord-Kamina) [inline data/config/schema.xml:145]: In English, punctuation goes inside the parentheses.

**Review by Lord-Kamina — CHANGES_REQUESTED** [review]

File: data/config/schema.xml (hunk 1 above)
Comment (Baklap4) [reply data/config/schema.xml:145]: That depends but in this case you're right
Will fix it tomorrow

Comment (Baklap4) [reply data/config/schema.xml]: Fixed

File: lang/fr.po [hunk 2]
```diff
@@ -100,16 +100,38 @@ msgstr "Mode karaoké"
...
+#. Difficulty
+#. Game difficulty.
+#. name: game/difficulty
+#. type: int
+#. value: 0
+#: ../data/config/schema.xml:57
+#, fuzzy
+#| msgid "Select difficulty"
+msgid "Difficulty"
+msgstr "Sélectionner la difficulté"
```
Comment (OznOg) [inline lang/fr.po:116]:
```suggestion
msgstr "Choisir la difficulté"
```

File: lang/fr.po [hunk 3]
```diff
@@ -100,16 +100,38 @@ msgstr "Mode karaoké"
...
+#. Difficulty
+#. Game difficulty.
+#. name: game/difficulty
+#. type: int
+#. value: 0
+#: ../data/config/schema.xml:58
+#, fuzzy
+#| msgid "Select difficulty"
+msgid "Game difficulty."
+msgstr "Sélectionner la difficulté"
```
Comment (OznOg) [inline lang/fr.po:127]:
```suggestion
msgstr "Choisir la difficulté"
```

File: lang/fr.po [hunk 4]
```diff
@@ -305,20 +331,38 @@ msgstr "Port de repli"
...
 "disponible. Prenez un un qui soit certainement libre."
 
+#. Note Graph Scaling
+#. Type of scaling the note graph.
+#. name: game/notegraphscalingmode
+#. type: int
+#. value: 0
+#: ../data/config/schema.xml:132
+msgid "Note Graph Scaling"
+msgstr ""
```
Comment (OznOg) [inline lang/fr.po:349]:
```suggestion
msgstr "Mise à l'échelle des notes"
```

File: lang/fr.po [hunk 5]
```diff
@@ -305,20 +331,38 @@ msgstr "Port de repli"
...
+
+#. Note Graph Scaling
+#. Type of scaling the note graph.
+#. name: game/notegraphscalingmode
+#. type: int
+#. value: 0
+#: ../data/config/schema.xml:133
+msgid "Type of scaling the note graph."
+msgstr ""
```
Comment (OznOg) [inline lang/fr.po:358]:
```suggestion
msgstr "Type de mise a l'échelle"
```

File: lang/fr.po [hunk 6]
```diff
@@ -345,34 +389,34 @@ msgstr "Batterie au clavier"
...
+#. Drums use Pro mode (toms are not cymbals).
 #. name: game/drum_promode
 #. type: bool
 #. value: true
-#: ../data/config/schema.xml:123
-msgid "Drums use Pro mode (toms are not cymbals)"
-msgstr ""
+#: ../data/config/schema.xml:145
+msgid "Drums use Pro mode (toms are not cymbals)."
+msgstr "Le batterie est en mode Pro (les toms ne sont pas de cymbales)."
```
Comment (OznOg) [inline lang/fr.po:412]:
```suggestion
msgstr "Le batterie est en mode Pro (les toms ne sont pas des cymbales)."
```

**Comment by Lord-Kamina** [comment]
I've updated the base string for the notegraph scaling option, as it wasn't very well written.
Some languages (from my very limited understanding of other languages) appeared to have a word-by-word translation of it, so you might want to revise it.

**Review by Lord-Kamina — APPROVED** [review]
LGTM

**Pull request merged**